atlite/resource.py

```diff
@@ -53,10 +54,12 @@ def get_windturbineconfig(turbine):
         return get_oedb_windturbineconfig(turbine[len("oedb:") :])
 
     if isinstance(turbine, str):
```

Comment:
We could enforce the user to provide a `Path` object if the configuration is supposed to be read from a local path (instead of asking for a `str`. At the moment, if a `Path` object is provided the method fails.

That would also allow for better overloading of the argument :)

```suggestion
    if isinstance(turbine, Path):
```